.github/pull_request_template.md

```diff
@@ -5,12 +5,12 @@
 ...
 
 ### Checklist before you submit for review
-- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version
+- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version if your PR is changing the API
 - [ ] Use `NoSpamLogger` for log lines that may appear frequently in the logs
-- [ ] Verify test results on Butler
-- [ ] Test coverage for new/modified code is > 80%
+- [ ] Verify Butler and Sonar checks are passing
 - [ ] Proper code formatting
 - [ ] Proper title for each commit staring with the project-issue number, like CNDB-1234
 - [ ] Each commit has a meaningful description
-- [ ] Each commit is not very long and contains related changes
-- [ ] Renames, moves and reformatting are in distinct commits
\ No newline at end of file
+- [ ] Each commit contains related changes
+- [ ] Renames, moves and reformatting are in distinct commits
```

Comment:
I would still drop this one, as I think most of the time it would generate more - artificially separated - work.

Reply:
I can drop it, but it is just a friendly reminder - personally I'd not start reviewing a PR which mixes renamings and major reformattings with the code changes - just with "how to write tests" - it is just sharing "how to"